test/Rules.js

```diff
@@ -41,12 +40,97 @@ describe("Rules", function() {
     assert.deepEqual(results.errors.all, ["person", "company"]);
   });
 
-  it("populates multiple outcomes into an array when [] is added", function() {
+
+  it("tests alternative outcome if rule validation comes back false", function () {
+    var config = {
+      "Person will be in house if person is tired or hungry": {
+        if: {
+          "person.tired": true
+        },
+        then: {
+          "person.location": "house"
+        },
+        otherwise: {
+          "person.location": "work"
+        }
+      }
+    };
+    var data = {
+      person: {
+        tired: false,
+        hungry: false
+      }
+    };
+
+    var rules = new Rules(config);
+    rules.run(data);
+
+    assert.equal(data.person.location, "work");
+  });
+
+  it("tests a series of if conditions (Array), as separate OR rules", function () {
+    var config = {
+      "Person will be in house if person is tired or hungry": {
+        if: [
+          {"person.tired": true},
+          {"person.hungry": true}
+        ],
+        then: {
+          "person.location": "house"
+        },
+        otherwise: {
+          "person.location": 'work'
+        }
+      }
+    };
+    var data = {
+      person: {
+        tired: false,
+        hungry: true
+      }
+    };
+
+    var rules = new Rules(config);
+    rules.run(data);
+
+    assert.equal(data.person.location, "house");
+  });
+
+  it("tests a series of if (OR) conditions, and applies `otherwise` outcome if all fail", function () {
+    var config = {
+      "Person will be in house if person is tired or hungry": {
+        if: [
+          {"person.tired": true},
+          {"person.hungry": true}
+        ],
+        then: {
+          "person.location": "house"
+        },
+        otherwise: {
+          "person.location": "work"
+        }
+      }
+    };
+    var data = {
+      person: {
+        tired: false,
+        hungry: false
+      }
+    };
+
+    var rules = new Rules(config);
+    rules.run(data);
+
+    assert.equal(data.person.location, "work");
+  });
+
+  it("populates multiple outcomes into an array when [] is added", function () {
     var config = {
       "Must be 21 or older": {
         if: {
-          "person.age": { lessThan: 21}
```

Comment:
There are still quite a few changes in the file that aren't really a part of the PR.  If you can remove those from the PR, then we're good to go!